docker-entrypoint.sh

```diff
@@ -103,22 +105,31 @@ docker_verify_minimum_env() {
 
 		EOWARN
 	fi
-	if [ -z "$POSTGRES_PASSWORD" ]; then
+	if [ -z "$POSTGRES_PASSWORD" ] && [ -z "$POSTGRES_DISABLE_PASSWORDS" ]; then
 		# The - option suppresses leading tabs but *not* spaces. :)
+		cat >&2 <<-'EOE'
+			Error: Database is uninitialized and superuser password is not specified.
+			       You must specify either POSTGRES_PASSWORD or POSTGRES_DISABLE_PASSWORDS
+			       Use "-e POSTGRES_PASSWORD=password" to set it in "docker run".
+		EOE
+		exit 1
+	fi
+	if [ -n "$POSTGRES_DISABLE_PASSWORDS" ]; then
 		cat >&2 <<-'EOWARN'
 			****************************************************
-			WARNING: No password has been set for the database.
+			WARNING: POSTGRES_DISABLE_PASSWORDS has been set.
 			         This will allow anyone with access to the
-			         Postgres port to access your database. In
-			         Docker's default configuration, this is
+			         Postgres port to access your database without
+			         a password, even if POSTGRES_PASSWORD is set.
```

Comment:
Perhaps this (or the `pg_setup_hba_conf` function) should also print what is added to `pg_hba.conf`?

Reply:
Maybe just a link to https://www.postgresql.org/docs/12/auth-trust.html ?